packages/rich-text-editor/src/EditableRichTextContent/EditableRichTextContent.tsx

```diff
@@ -46,13 +46,12 @@ export const EditableRichTextContent: React.VFC<
         className={classnames(styles.editableContainer, classNameOverride)}
         {...restProps}
       >
-        <VisuallyHidden>
```

Comment:
I'll also need to remove the import of this if we go this route but holding off until I can get feedback :)